src/NuGet.Core/NuGet.ProjectModel/DependencyGraphSpec.cs

```diff
@@ -300,7 +300,7 @@ public void Save(Stream stream)
 
         public string GetHash()
         {
-            using (var hashFunc = new Sha512HashFunction())
+            using (var hashFunc = new FnvHash64Function())
```

Comment:
It would be a ton of work to have an escape hatch other than env var to bring back the old behavior.  Should we check for an env var to bring back the SHA512 hash just in case?

Reply:
I've added an escape hatch `NUGET_ENABLE_LEGACY_DGSPEC_HASH_FUNCTION=true`